script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt

```diff
@@ -277,67 +289,67 @@ It's as if she's in a entirely different dimension.
 [sha:fad3f380559bb2665e20e37f005480be249b1b9b]{
 -- Page 276, Offset 32431.
 -- 知らず、シエルは呼吸を再開していた。
-Ciel unconsciously resumes breathing again.
+Without realizing it, Ciel begins breathing again.
 }
 [sha:4716620504c9d2ee454c1a6ef8e655acb1811e15]{
 -- Page 276, Offset 32432.
 -- ロアの事も、志貴の事も、アルクェイドの事も、空白のように思考から消え去った。
-The things about Roa, Shiki, and Arcueid, disappear from her thoughts like a void.
+All thoughts relating to Roa, Shiki or Arcueid vanish from her mind.
```

Comment:
any thought relating to
vanishes from

Reply:
I feel like I prefer `all` here, though it's not ahead by a crazy margin in ngrams. Could maybe simplify a bit here to 
> All thoughts of ... 

though

Reply:
current line is fine if you swap "or" for "and" if you prefer this structure.